uxarray/grid.py

```diff
@@ -528,6 +540,35 @@ def _build_edge_node_connectivity(self):
         setattr(self, "Mesh2_edge_nodes", self.ds['Mesh2_edge_nodes'])
         setattr(self, "nMesh2_edge", edge_nodes_unique.shape[0])
 
+    def _build_face_edges_connectivity(self):
+        """Constructs the UGRID connectivity variable (``Mesh2_face_edges``)
+        and stores it within the internal (``Grid.ds``) and through the
+        attribute (``Grid.Mesh2_face_edges``)."""
+        if ("Mesh2_edge_nodes" not in self.ds or
+                "inverse_indices" not in self.ds['Mesh2_edge_nodes'].attrs):
+            self._build_edge_node_connectivity(repopulate=True)
+
+        inverse_indices = self.ds['Mesh2_edge_nodes'].inverse_indices
+        inverse_indices = inverse_indices.reshape(self.nMesh2_face,
+                                                  self.nMaxMesh2_face_nodes)
+        mesh2_face_edges = inverse_indices  # We only need to store the edge index
```

Comment:
This line isn't necessary, we can simply pass through `inverse_indices` into the `data=` in our Data Array